frontend/packages/kubevirt-plugin/integration-tests-cypress/utils/const/index.ts

```diff
@@ -61,7 +62,7 @@ export enum VM_STATUS {
   Starting = 'Starting',
   Provisioning = 'Provisioning',
   Paused = 'Paused',
-  Migrating = 'Migration',
+  Migrating = 'Migrati',
```

Comment:
`Migrati` is a typo.

Reply:
No, it's intentional, the strings differ in different perspectives. 'Migrating' in Admin, but 'Migration' in Topology. The rest of strings are identical, so I shortened the string to match both cases.